move_base_msgs/CMakeLists.txt

```diff
@@ -1,25 +1,18 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(move_base_msgs)
 
-find_package(catkin REQUIRED
-  COMPONENTS
-    message_generation
-    actionlib_msgs
-    geometry_msgs
-)
+find_package(ament_cmake REQUIRED)
+find_package(builtin_interfaces REQUIRED)
```

Comment:
I don't think this package directly depends on `builtin_interfaces` or `std_msgs`, I think both could be removed from the `CMakeLists.txt` and `package.xml`.

Reply:
Fair enough, I usually add those defensively -- will remove.